src/applications/ivc-champva/10-10D/pages/ApplicantAddressPage.jsx

```diff
@@ -1,11 +1,9 @@
 import React, { useState, useEffect } from 'react';
```

Comment:
Removed radio button and added certifier address + sponsor address to list of values we can choose from.